spec/doctor/dnsmasq_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "spec_helper"
+require "./lib/doctor/dnsmasq"
+
+describe Doctor::Dnsmasq do
+
+  context "when dnsmasq is not installed" do
+    it "should tell me to install dnsmasq if it is not yet installed" do
+      allow(subject).to receive(:system).with("which dnsmasq 1>/dev/null").and_return(false)
+      subject { described_class.new }
+      expect(subject.call).to include("Dnsmasq not found")
+    end
+  end
+
+  context "when dnsmasq is installed" do
+    it "should report that dnsmasq is installed if it is" do
+      subject { described_class.new }
```

Comment:
You could move this line to the top of the `describe` since it's the same in each `it` block.

Reply:
good idea - done that too